scripts/easyarty/functions/fn_setTargetPos.sqf

```diff
@@ -0,0 +1,17 @@
+openMap [true, true];
+
+hint "Click on the map to place a target";
+
+private _mapClickEh = addMissionEventHandler ["MapSingleClick", {
+	params ["", "_pos"];
+	deleteMarker "target_loc";
+	_marker = createMarkerLocal ["target_loc", _pos];
+	_marker setMarkerType "HD_DOT";
```

Comment:
this will make the marker global, i doubt you want this. Also use prefixing for the marker name.